### PR TITLE
docs: change ctr fuel pump to jet pump

### PR DESCRIPTION
**Pull request by Sleinmaster**
## Summary

changed description for the ctrl pump buttons from the CEOs traditional pumps to the NEOs jet pumps.
### Location

/pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel/?h=fuel#ctr-tk-pump-1-2

Discord username (if different from GitHub): Slein

### CODE REVIEW

**Comment by Sleinmaster** [comment]
@donstim do the ctr valve buttons still show faults? If yes, for what conditions?

**Comment by donstim** [comment]
> @donstim do the ctr valve buttons still show faults? If yes, for what conditions?

No, they shouldn't anymore. They should only show on or off. If the mode sel is un Auto position, on means the valves are automatically controlled. If the mode sel is in manual, then putting the button to on opens the valve and putting it to off closes the valve.

The mode sel can be auto, man(ual), or fault. The fault light illuminates and ECAM caution comes on if the center tank has more than 250 kg of fuel and the left or right inner tank has less than 5000 kg of fuel and the mode sel is in auto. (Actually, I don't remember if that fault was implemented and I am traveling so I can't check.) @tracernz would know. 

Since I am traveling I haven't kept up with what is being changed in the documentation. But as far as the variable changes for the valves vs pumps, there are now 2 valves in serial on each side (left and right) instead of the pumps. Tracernz had to do that to get the manual mode to work right (that is, to prevent the MSFS triggers for automatically opening the valves to not cause fuel to transfer if the mode sel is in manual and the valve pushbutton is in the closed position. Indexes for the left side valves are 9 and 11, right side ones are 10 and 12. (Maybe that doesn't need to be identified, though, since valves 9 and 10 are set by the position of the mode sel switch and valves 11 and 12? If the mode sel switch is in auto, then valves 9 and 10 are open. The amount of fuel in the center and inner tanks will set the position of valves 11 and 12. If the mode sel switch is off, then valves 9 and 10 will be closed unless valves 11 and 12 are opened manually.)

**Comment by Sleinmaster** [comment]
For reference https://github.com/flybywiresim/a32nx/pull/7817

**Comment by Sleinmaster** [comment]
@donstim @tracernz 
If I read the code correctly valve 9 and 10 are what the users have to worry about and what's opened by the buttons and valve 11 and 12 are handled by the plane, correct?
Screenshot_20230415-113917

**Comment by Sleinmaster** [comment]
@donstim 
If I read it right, this code part is for the fault, so it should be implemented.
image

**Comment by tracernz** [comment]
> If I read the code correctly valve 9 and 10 are what the users have to worry about and what's opened by the buttons and valve 11 and 12 are handled by the plane, correct?

The user shouldn't need to worry about any valve indices, just press the "pump" buttons on the overhead to disable transfer.

> @donstim If I read it right, this code part is for the fault, so it should be implemented. image

Yeah, that code was already there.

**Comment by Sleinmaster** [comment]
> > If I read the code correctly valve 9 and 10 are what the users have to worry about and what's opened by the buttons and valve 11 and 12 are handled by the plane, correct?
> 
> The user shouldn't need to worry about any valve indices, just press the "pump" buttons on the overhead to disable transfer.
> 
> > @donstim If I read it right, this code part is for the fault, so it should be implemented. image
> 
> Yeah, that code was already there.

In normal operation sure, but if the flight deck api is used, as far as I understood the events needed changed from FUELSYSTEM_PUMP_TOGGLE 1 and 4 to FUELSYSTEM_VALVE_TOGGLE 9 and 10.

image

**Comment by Sleinmaster** [comment]
I think this should reflect the new behaviour:
image

**Comment by donstim** [comment]
I think that's correct, except that I should have been more specific about when the valves automatically close for the condition "when the center tank is empty." They actually don't close until 5 minutes after the center tank low fuel level (130 kg) is reached.

**Comment by donstim** [comment]
> > > If I read the code correctly valve 9 and 10 are what the users have to worry about and what's opened by the buttons and valve 11 and 12 are handled by the plane, correct?
> > 
> > 
> > The user shouldn't need to worry about any valve indices, just press the "pump" buttons on the overhead to disable transfer.
> > > @donstim If I read it right, this code part is for the fault, so it should be implemented. image
> > 
> > 
> > Yeah, that code was already there.
> 
> In normal operation sure, but if the flight deck api is used, as far as I understood the events needed changed from FUELSYSTEM_PUMP_TOGGLE 1 and 4 to FUELSYSTEM_VALVE_TOGGLE 9 and 10.
> 
> image

For this one, there is no direct replacement in this manner if I understand how the api works. Additional logic is embedded in the buttons other than just opening or closing the valves. For example, if the mode sel is in manual, putting an overhead pump (now valve) position to open will open that valve, and fuel will transfer until the button is pushed to close the valve. If you simply open the valve via api by "opening" index 9 (or 10), the valve will close again when the automatic "close" trigger condition is reached, which would be incorrect behavior. If the api used indexes 11 and 12 instead of 9 and 10, then fuel would transfer whenever these valves are open even if the mode sel is in auto, which again is incorrect behavior. (Note: the valves work in parallel, not serially as I said previously.)

Disregard this if I am wrong about how the api works.

**Comment by Sleinmaster** [comment]
> > > > If I read the code correctly valve 9 and 10 are what the users have to worry about and what's opened by the buttons and valve 11 and 12 are handled by the plane, correct?
> > > 
> > > 
> > > The user shouldn't need to worry about any valve indices, just press the "pump" buttons on the overhead to disable transfer.
> > > > @donstim If I read it right, this code part is for the fault, so it should be implemented. image
> > > 
> > > 
> > > Yeah, that code was already there.
> > 
> > 
> > In normal operation sure, but if the flight deck api is used, as far as I understood the events needed changed from FUELSYSTEM_PUMP_TOGGLE 1 and 4 to FUELSYSTEM_VALVE_TOGGLE 9 and 10.
> > image
> 
> For this one, there is no direct replacement in this manner if I understand how the api works. Additional logic is embedded in the buttons other than just opening or closing the valves. For example, if the mode sel is in manual, putting an overhead pump (now valve) position to open will open that valve, and fuel will transfer until the button is pushed to close the valve. If you simply open the valve via api by "opening" index 9 (or 10), the valve will close again when the automatic "close" trigger condition is reached, which would be incorrect behavior. If the api used indexes 11 and 12 instead of 9 and 10, then fuel would transfer whenever these valves are open even if the mode sel is in auto, which again is incorrect behavior. (Note: the valves work in parallel, not serially as I said previously.)
> 
> Disregard this if I am wrong about how the api works.

https://github.com/flybywiresim/a32nx/pull/7817/files#diff-45c9eb66ea69328e5d702c11997668a80ceb79d5d4e424861f8e3477e5495166

I think the FUELSYSTEM_VALVE_TOGGLE only changes the value of the switch, but I'm not entirely sure how to read that code.
Might try to test it later.

**Comment by Sleinmaster** [comment]
Also not sure if using the FUELSYSTEM_PUMP_TOGGLE in the <Template Name="FBW_JET_PUMP"> is a bug.

**Comment by donstim** [comment]
I described the potential problem with the api incorrectly and deleted my comment, but I see I didn't do it quickly enough! The problem is that if the mode sel is in auto, but then the user opens valve 9 or 10 through the api, fuel will transfer and not be auto controlled.

File: docs/pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel.md [hunk 1]
```diff
@@ -42,29 +42,23 @@ Both engines can be fed from one side, or both sides can feed only one engine.
 ### MODE SEL
 
 - AUTO:
-    - Automatic control.
-        - Runs 2 minutes at engine start.
-        - Outside the engine start sequence, the pumps activate while slats are retracted.
-        - After the center tank low level is reached, they deactivate after 5 minutes.
+    - Automatic control of transfer valves from center to inner tanks, if the CTR TK PUMP are set to ON/AUTO
+        - Valves open if inner tank level is 500 kg below inner tank capacity.
+        - Valves close when the innter tank is full or 5 minutes after the center tank low fuel level (130 kg) is reached.
```
Comment (Valastiri) [inline docs/pilots-corner/a32nx-briefing/flight-deck/ovhd/fuel.md]:
```suggestion
        - Valves close when the inner tank is full or 5 minutes after the center tank low fuel level (130 kg) is reached.
```

**Review by Valastiri — COMMENTED** [review]
spelling only

**Pull request merged**